projects/download GeeksForGeeks articles/downloader.py

```diff
@@ -0,0 +1,49 @@
+# !/usr/bin/env python
+from selenium import webdriver
+import json
+import requests
+
+
+# article url
+# URL = "https://www.geeksforgeeks.org/what-can-i-do-with-python/"
+
+
+def download_article(URL):
+    # chrome options settings
+    chrome_options = webdriver.ChromeOptions()
+    settings = {
+        "recentDestinations": [
+            {"id": "Save as PDF", "origin": "local", "account": ""}
+        ],
+        "selectedDestinationId": "Save as PDF",
+        "version": 2,
+    }
+    prefs = {
+        "printing.print_preview_sticky_settings.appState": json.dumps(settings)
+    }
+    chrome_options.add_experimental_option("prefs", prefs)
+    chrome_options.add_argument("--kiosk-printing")
+    chrome_options.binary_location = (
```

Comment:
don't store any static path

use following code this will automatically install the web-driver if not installed
```python
from webdriver_manager.firefox import GeckoDriverManager
def get_driver():
    options = Options()
    options.headless = True
    driver = webdriver.Firefox(
        executable_path=GeckoDriverManager().install(), options=options)
    return driver

Reply:
@chavarera removed dependency of static location.